nanome/api/schemas/structure_schemas.py

```diff
@@ -61,12 +58,13 @@ def make_atom(self, data, **kwargs):
         return new_obj
 
 
-class BondSchema(StructureSchema):
-    atom1 = AtomSchema(only=('index',))
-    atom2 = AtomSchema(only=('index',))
+class BondSchema(Schema):
+    index = fields.Integer(default=-1)
+    atom1 = fields.Nested(AtomSchema(only=('index',)))
```

Comment:
@ajm13 question here; do you think we need to add the full atom here, or can we get away with just the index? The full atom data get stored in residue.atoms, so its a lot of redundancy writing all that here. But I'm not sure how NTS handles that

Reply:
Just index is best imo. Internally, `_bond_serializer.py` uses `_atom_serializer_id.py` which just serializes the index of the atom in the `"Atom"` payload array. So similar to Nanome index, but not exactly the same.